vsts/pipelines/templates/_buildTemplate.yml

```diff
@@ -198,12 +198,6 @@ steps:
       BuildDropPath: '$(Build.ArtifactStagingDirectory)'
       AdditionalComponentDetectorArgs: '--DirectoryExclusionList **/SampleApps/**'
 
-- script: |
-    echo "Writing Debian-specific runtime image tags to known file"
```

Comment:
Just mentioning this for other people reviewing and being confused on why this part is being removed: this logic existed in the build case but @cormacpayne determined it wasn't used there. Removing it makes sense so that we don't keep things around that are not used.